foreshadow/core/preparerstep.py

```diff
@@ -249,6 +247,28 @@ def parallelize_mapping(self, column_mapping):
             else:  # could be separated and parallelized
                 final_mapping[group_number] = transformer_list
                 parallelized[group_number] = True
+            # pipeline = column_mapping[group_number]
```

Comment:
What was the issue here?

Reply:
Not sure, I'll take another look, I might need to just remove the comment